### ISSUE TITLE
Reactive Endpoints WebSocket URL doesn't consider spring-boot configured spring.servlet.context-path

### ISSUE THREAD

**Issue by hu-chia** [vaadin/hilla#1229]
### Description of the bug 

I wrote an application with hiila on springboot, and for reverse proxy convenience I configured the springboot property `server.servlet.context-path`, for now it works fine. but when I implemented a reactive endpoint follow by [this documentation](https://hilla.dev/docs/react/guides/reactive-endpoints), it not working. I believe this is a bug about websocket in hilla code generator,.

### Minimal reproducible example

https://github.com/hu-chia/hilla-issue-reproduce

### Expected behavior

when I use reactive endpoints, the request url should has the correct prefix.

### Actual behavior

I got 404 because the frontend always request `ws://127.0.0.1:8080/HILLA/push/*`.

```
WebSocket connection to 'ws://localhost:8080/HILLA/push?X-Atmosphere-tracking-id=0&X-Atmosphere-Framework=3.1.4-javascript&X-Atmosphere-Transport=websocket&X-Atmosphere-TrackMessageSize=true&Content-Type=application/json;%20charset=UTF-8&X-atmo-protocol=true&X-CSRF-Token=70552295-ddfe-4b49-b0c8-0c2359d10c4c' failed: Error during WebSocket handshake: Unexpected response code: 404
```

**Comment by hu-chia** [comment]
is there any workaround?

**Comment by Artur-** [comment]
I think the problem is that the URL for push is set to `/HILLA/push` instead `HILLA/push`. The latter would always be relative to the context path.

**Comment by Artur-** [comment]
as the fix is here https://github.com/vaadin/hilla/pull/1232/files#diff-2eb5ba03ac65fe3182c254a453e2264948c09b780eccbe04859b8417a75288faR141 I don't see any other workaround than using `patch-package` if really needed

**Comment by hu-chia** [comment]
@Artur- Can I reopen this issue because I noticed those fixes was not actually released with Hilla 2.1.9?

https://github.com/vaadin/hilla/blob/c0556f6bebac25336152e8b9ed62a851ad42fc0a/packages/ts/hilla-frontend/src/FluxConnection.ts#L76

**Comment by Artur-** [comment]
The fix is in 2.2.0
